hw/ip/aes/pre_sca/alma_post_syn/load_env.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This file needs to be sources into shell as follows:
+# `source load_env.sh`
+
+# The top folder for opentitan repository
+export REPO_TOP=~/Documents/opentitan
+# The top folder for the private ASIC repo
+export ASIC_TOP=~/Documents/asic
+# The top folder for the CocoAlma repo
+export ALMA_TOP=~/Documents/coco-alma
+# The output folder for temporary files created during the flow
+export OUTPUT_DIR=/tmp/alma_tmp
+
+# The following paths should not need any modification, however patch and netlist files need to be
+# manually placed in the respective folders (since they are not part of the public repo).
+export SIM_MODEL_FILE=${ASIC_TOP}/vendor/std_cell/verilog/ts40n8ksdst.v
+export AES_SBOX_NETLIST_FILE=${ASIC_TOP}/alma/aes_sbox_SecSBoxImpl4.gv
+export SIM_PATCH_FILE=${ASIC_TOP}/alma/techlib_patch.v
+export MODIFIED_TECHLIB_FILE=${OUTPUT_DIR}/modified_cell_library.v
```

Comment:
Same here, I think the script should check that the files exist. The errors produced downstream by missing files might be harder to debug.